src/zendesk/client.ts

```diff
@@ -0,0 +1,14 @@
+import zendesk from 'node-zendesk';
+import { Client, ClientOptions } from 'node-zendesk'
+
+function newClient(username: string, token: string, remoteUri: string): Client {
```

Comment:
```suggestion
export default function newClient(username: string, token: string, remoteUri: string): Client {
```

You can also get rid of the export on line 14

Reply:
done.